tests/integration/cases/check_vsa_sh_fail/test.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2024 - 2024, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+description: |
+  Running check_vsa.sh with invalid arguments and VSA.
+
+tags:
+- macaron-general-scripts
+
+steps:
+- name: Copy the check_vsa.sh from the source repo to ./output which will eventually be cleaned up.
+  kind: shell
+  options:
+    cmd: ./setup.sh
+- name: Download JAR file
+  kind: shell
+  options:
+    cmd: curl -o ./output/micronaut-core-4.5.3-oracle-00001.jar https://maven.oracle.com/public/io/micronaut/micronaut-core/4.5.3-oracle-00001/micronaut-core-4.5.3-oracle-00001.jar
+- name: Check a failing VSA.
+  kind: shell
+  options:
+    cmd: ./output/check_vsa.sh --artifact-path ./output/micronaut-core-4.5.3-oracle-00001.jar --vsa-path ./resources/vsa-micronaut-core-4.5.3-invalid.intoto.jsonl --purl pkg:maven/io.micronaut/micronaut-core@4.5.3-oracle-00001?type=jar
+  expect_fail: true
+- name: Check invalid arguments.
+  kind: shell
+  options:
+    cmd: ./output/check_vsa.sh --artifact-path ./output/micronaut-core-4.5.3-oracle-00001.jar --vsa-path ./resources/vsa-micronaut-core-4.5.3-invalid.intoto.jsonl
+  expect_fail: true
+- name: Check invalid arguments.
+  kind: shell
+  options:
+    cmd: ./output/check_vsa.sh --artifact-path ./output/micronaut-core-4.5.3-oracle-00001-invalid.jar --vsa-path ./resources/vsa-micronaut-core-4.5.3-invalid.intoto.jsonl
+  expect_fail: true
+- name: Check invalid arguments.
+  kind: shell
+  options:
+    cmd: ./output/check_vsa.sh --artifact-path ./output/micronaut-core-4.5.3-oracle-00001.jar --vsa-path ./resources/vsa-micronaut-core-4.5.3-invalid.intoto.jsonl --purl pkg:maven/io.micronaut/micronaut-core@4.5.3-oracle-00001?type=pom
+  expect_fail: true
```

Comment:
It would be good if we add which argument is invalid in each of these steps.

Reply:
See commit [32e104c](https://github.com/oracle/macaron/pull/858/commits/32e104c12ea49dec6675910dc6e9a3ea59baf4d9)